### PR TITLE
Fixed empty chat container inverted

### PR DESCRIPTION
**Pull request by iqbalfaisal**
Fixed issue #1557 by changing the container of  renderChatEmpty in MessageContainer.tsc file.

### CODE REVIEW

File: src/MessageContainer.tsx [hunk 1]
```diff
@@ -306,7 +310,11 @@ export default class MessageContainer<
 
   renderChatEmpty = () => {
     if (this.props.renderChatEmpty) {
-      return this.props.renderChatEmpty()
+      return (
+        <View style={styles.emptyChatContainer}>
```
Comment (xcarpentier) [inline src/MessageContainer.tsx]: Hi @iqbalfaisal,
Thanks!
You have to be sure you are not in inverted mode before applying the style.
You should find prop `inverted` here, no?

Comment (iqbalfaisal) [reply src/MessageContainer.tsx]: Thanks @xcarpentier 
I have completed this PR. Please review my changes and merge to master.

**Comment by xcarpentier** [comment]
No sorry too much change involved here, I didn't realize that.

**Comment by iqbalfaisal** [comment]
> 
> 
> No sorry too much change involved here, I didn't realize that.
can mention them ?

**Comment by iqbalfaisal** [comment]
@xcarpentier Please mentions them.

**Comment by iqbalfaisal** [comment]
@xcarpentier  i have formatted the code that why you notices that too many changes

**Comment by xcarpentier** [comment]
If so please keep the same formatting with prettier, thanks

**Comment by iqbalfaisal** [comment]
@xcarpentier in last commit i restore it

**Comment by iqbalfaisal** [comment]
@xcarpentier DONE

File: src/MessageContainer.tsx [hunk 2]
```diff
@@ -59,92 +52,184 @@ const styles = StyleSheet.create({
     shadowRadius: 1,
   },
 })
-
-export interface MessageContainerProps<TMessage extends IMessage> {
```
Comment (xcarpentier) [inline src/MessageContainer.tsx]: ???

**Comment by xcarpentier** [comment]
Still not good...
Sorry, but why did you remove the typescript props definition?

**Comment by iqbalfaisal** [comment]
@xcarpentier Sorry Brother it's wrong with my prettier config file but now it's solved. You can check it.

**Comment by xcarpentier** [comment]
I guess the is gone no `emptyChatContainer`?

**Comment by iqbalfaisal** [comment]
@xcarpentier Sorry my fault. I forgot to add style. Now i have added the  `emptyChatContainer` style

**Comment by iqbalfaisal** [comment]
@xcarpentier are you there???

**Review by JoeMartin2001 — COMMENTED** [review]
Hi. How about transform **scaleX**?

**Pull request merged**